models/lobby.go

```diff
@@ -260,6 +260,19 @@ func (lobby *Lobby) UnreadyPlayer(player *Player) *helpers.TPError {
 	return nil
 }
 
+func (lobby *Lobby) RemoveUnreadyPlayers() {
+	var slots []*LobbySlot
+	db.DB.Where("lobby_id = ?", lobby.ID).Find(slots)
+
+	for _, slot := range slots {
+		if !slot.Ready {
+			db.DB.Where("player_id = ? AND lobby_id = ?", slot.ID, lobby.ID).Delete(&LobbySlot{})
```

Comment:
If we're doing it this way, you should change slot.ID to slot.PlayerId. In the future I'd like to change methods like lobby.UnreadyPlayer to accept an id instead of a player object so that we could do `lobby.UnreadyPlayer(slot.PlayerId) `in places like this.